homeassistant/components/tractive/__init__.py

```diff
@@ -124,10 +128,16 @@ async def _listen(self):
                     if event["message"] == "activity_update":
                         self._send_activity_update(event)
                     else:
-                        if "hardware" in event:
+                        if "hardware" in event and self._last_hw_time != event[
+                            "hardware"
+                        ].get("time"):
```

Comment:
The same package is often received several times. This to only update the sensors for new values